docs/sql/statements/start-transaction.rst

```diff
@@ -0,0 +1,46 @@
+.. highlight:: psql
+
+.. sql-start-transaction:
+
+=====================
+``START TRANSACTION``
+=====================
+
+This is PostgreSQL compatibility feature that the statement is implemented but
+will be silently ignored.
```

Comment:
```suggestion
CrateDB accepts the ``START TRANSACTION` statement for compatibility with the
:ref:`PostgreSQL wire protocol <postgres_wire_protocol>`. However, CrateDB does
not support transactions and will silently ignore this statement.

.. SEEALSO::

    :ref:`Appendix: SQL compatibility <crate_standard_sql>`
```

cool! thanks for updating this bit

I took a second look at this file with your new changes and I think we could improve it by moving the bulk of the info to this para

my suggested change:

- reword a little so CrateDB is more of an active participant (i.e., the noun doing the verbs)
- made `START TRANSACTION` explicit in the first instance (and "this statement" in the second instance)
- expanded and link "PostgreSQL wire protocol"
- moved up your link to `crate_standard_sql` from below (good addition!) and turned it into a `SEEALSO` block with explicit link text. because this is a `SEEALSO` block, I added an "Appendix: " prefix to identify which section of the docs this page belongs to